spec/features/venues/view_venue_details_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+feature 'View details of the venue' do
```

Comment:
In terms of using terminology that's widely used, calling this a "Venue show page" would also be accurate and possibly more explicit for future devs